envoy-control-tests/src/main/resources/lua_spec/ingress_handler_spec.lua

```diff
@@ -0,0 +1,227 @@
+require("ingress_handler")
```

Comment:
1. How do I run this test manually on my host machine? How do I debug it? Can I debug it inside a testcontainer?

An instruction in docs would be great.

2. How do these tests show up on CI?

Reply:
1. You just run https://github.com/allegro/envoy-control/blob/fba98b86fb640bf91c019174647cb93794145d0d/envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/LuaTest.kt class - it is a regular Junit test. You can do it from gradle or from Intellij just clicking on the green arrow :)

Reply:
> How do I debug it? Can I debug it inside a testcontainer?
There is no a debugger for Lua that works as we are used to, as far as I know. Especially remote debugger which would work in testcontainers.
If you look closely, I added one usefull debugging command in a comment at the end of the test file